main/src/main/resources/org/clulab/numeric/HOLIDAY.tsv

```diff
@@ -0,0 +1,49 @@
+#
+# List of holidays in the United States extracted from the Jollyday api.
+# This list can be obtained with the following snippet:
+#
+#   import de.jollyday.{HolidayManager, HolidayCalendar, ManagerParameters}
+#   import collection.JavaConverters._
+#   val holidayManager = HolidayManager.getInstance(ManagerParameters.create(HolidayCalendar.UNITED_STATES))
+#   holidayManager.getCalendarHierarchy.getChildren.keySet.asScala.
+#                   map(holidayManager.getHolidays(2021, _).asScala).flatten.map(_.getDescription)
+#
+#
+Lincoln's Birthday
+Easter Monday
+Washington's Birthday
+Labour day
```

Comment:
Spelt "Labor" around here ;-) but maybe we can't correct their list.

Reply:
We could use the "comment" trick I mention above. We would use their spellings as canonical forms but we could are our own alternatives:
```
Labour day // Labour day
Labor day // Labour day
```